### ISSUE TITLE
Running pre-commit on windows changes the symlinks in docs

### ISSUE THREAD

**Issue by Treesarj** [EMMC-ASBL/oteapi-core#129]
pre-commit removes the symlink nature of CHANGELOG.md and LICENSE.md under docs.

check commit: https://github.com/EMMC-ASBL/oteapi-core/pull/128/commits/caf7353a3bce2c5733e3623d96c889646c93f32e
check workflow: https://github.com/EMMC-ASBL/oteapi-core/runs/5551664763

**Comment by CasperWA** [comment]
I'll just underline that this is **only** the case when developing in Windows.

**Comment by jesper-friis** [comment]
It still looks like a real issue to me - it will hit us if symlinks suddenly are converted to files just because a Windows user have been pushing. 

Windows now has support for symlinks, but unfortunately it requires some local configuration on the Windows machine. 

See also: 
- https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201 
- https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows

**Comment by CasperWA** [comment]
Indeed, this is a real issue. It can be circumvented by using the Sphinx-framework for documentation instead of MkDocs, however. Or simply not use symlinks in the `docs` folder, but that would require more overhead.
